upup/models/cloudup/resources/addons/networking.cilium.io/k8s-1.16-v1.15.yaml.template

```diff
@@ -1773,69 +2033,75 @@ spec:
           timeoutSeconds: 3
         readinessProbe:
           httpGet:
-            host: '{{- if IsIPv6Only -}}::1{{- else -}}127.0.0.1{{- end -}}'
+            host: "{{- if IsIPv6Only -}}::1{{- else -}}127.0.0.1{{- end -}}"
             path: /healthz
             port: 9234
             scheme: HTTP
           initialDelaySeconds: 0
           periodSeconds: 5
           timeoutSeconds: 3
           failureThreshold: 5
-        terminationMessagePolicy: FallbackToLogsOnError
         volumeMounts:
-        - mountPath: /tmp/cilium/config-map
-          name: cilium-config-path
+        - name: cilium-config-path
+          mountPath: /tmp/cilium/config-map
           readOnly: true
 {{- if .EtcdManaged }}
-        - mountPath: /var/lib/etcd-config
-          name: etcd-config-path
+        - name: etcd-config-path
+          mountPath: /var/lib/etcd-config
           readOnly: true
-        - mountPath: /var/lib/etcd-secrets
-          name: etcd-secrets
+        - name: etcd-secrets
+          mountPath: /var/lib/etcd-secrets
           readOnly: true
 {{- end }}
+        securityContext:
+          allowPrivilegeEscalation: false
+          capabilities:
+            drop:
+            - ALL
+        terminationMessagePolicy: FallbackToLogsOnError
       hostNetwork: true
       restartPolicy: Always
       priorityClassName: system-cluster-critical
-{{ if ContainerdSELinuxEnabled }}
-      securityContext:
-        seLinuxOptions:
-          type: spc_t
-          level: s0
-{{ end }}
-      serviceAccount: cilium-operator
-      serviceAccountName: cilium-operator
+      serviceAccountName: "cilium-operator"
+      automountServiceAccountToken: true
+      # In HA mode, cilium-operator pods must not be scheduled on the same
+      # node as they will clash with each other.
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+          - labelSelector:
+              matchLabels:
+                io.cilium/app: operator
+            topologyKey: kubernetes.io/hostname
+      nodeSelector:
+        kubernetes.io/os: linux
       tolerations:
-      - operator: Exists
-      topologySpreadConstraints:
```

Comment:
should we keep the topology spread constraints?

Reply:
Official chart does it with `podAntiAffinity`. I want to not diverge, too hard to update in the future.